spec/purgatory_spec.rb

```diff
@@ -237,6 +237,17 @@
       end
     end
 
+    context "approving a performable method" do
+      before do
+        create_method_call_purgatory
+        @purgatory.soul.stub(:send).and_return(false)
+      end
+
+      it "should fail when performable method returns false" do
+        @purgatory.approve!(user2).should be_false
```

Comment:
We should also test that the purgatory wasn't marked as approved (shouldn't have an `approved_at` or `approved_by`)